carts/src/routes/index.ts

```diff
@@ -0,0 +1,61 @@
+import express from "express";
+const routes = express.Router();
+const cartItems = (
+  id: number,
+  product: string,
+  price: number,
+  quantity: number
+) => ({
+  id,
+  product,
+  price,
+  quantity,
+});
+const albums = [
+  cartItems(1, "Love Angel Music Baby", 12.99, 5),
+  cartItems(2, "Paradise", 9.99, 3),
+  cartItems(3, "Tako Tsubo", 8.99, 7),
+];
+routes.get("/", (req, res) => {
+  res.json("hello world");
+});
+routes.get("/cart-items", (req, res) => {
+  res.status(200).json(albums);
+});
+routes.get("/cart-items/:id", (req, res) => {
+  const itemId = +req.params.id;
+  const item = albums.find((item) => item.id === itemId);
+  if (item) {
+    res.status(200).json(item);
+  } else {
+    res.status(404).send("ID Not Found");
+  }
+});
+routes.post("/cart-items", (req, res) => {
+  albums.push(
+    cartItems(req.body.id, req.body.product, req.body.price, req.body.quantity)
```

Comment:
We want to create the ID on the back end instead of accepting an ID sent from the front end. When we create it on the back end we can be sure that we don't end up with duplicate IDs. 

It is common to use [a package like uuid](https://www.npmjs.com/package/uuid) to autogenerate unique IDs.